pint/models/parameter.py

```diff
@@ -321,21 +479,59 @@ def __init__(self, name=None, value=None, description=None, units='rad',
             'deg': (u.deg, 'deg', '%.15fdeg'),
         }
```

Comment:
I like how you are dealing with the various units using this dictionary. But is 'separator' the right name of this dictionary? When I think of a separator, I think of things like the colon in 'h:m:s'.

How about 'unit_operator' or something like that? (I don't have much inspiration, sorry)